tools/travis/py3k

```diff
@@ -81,19 +83,21 @@ if git diff | grep "^+ " | grep -v '[ ]*for' | grep -v [.]join | grep -v from_it
     echo
     echo "That was a lot of text!  Read this output from the top for more readable details."
     git reset --hard >/dev/null
-    failed=1
+    warn=1
```

Comment:
In other words, the intent of this big `if` clause is to filter out the false positives so we only fail if there's actually something to be changed